### PR TITLE
Linted/Migrated AreaSeries

### PR DESCRIPTION
**Pull request by bre1470**

### CODE REVIEW

File: ts/Series/Area/AreaSeries.ts [hunk 1]
```diff
@@ -27,11 +27,7 @@ import Color from '../../Core/Color/Color.js';
 const { parse: color } = Color;
 import LegendSymbol from '../../Core/Legend/LegendSymbol.js';
 import SeriesRegistry from '../../Core/Series/SeriesRegistry.js';
-const {
-    seriesTypes: {
-        line: LineSeries
-    }
-} = SeriesRegistry;
+const { seriesTypes: { line: LineSeries } } = SeriesRegistry;
```
Comment (TorsteinHonsi) [inline ts/Series/Area/AreaSeries.ts:30]: What is the rationale behind this change? This is a pattern we have never used before. In fact, we have consistently broken up deep objects into several lines... Should deconstructors be handled differently from object literals?

Comment (bre1470) [reply ts/Series/Area/AreaSeries.ts:30]: Looking at this, it shouldn't be a deconstruct at all.

Comment (bre1470) [reply ts/Series/Area/AreaSeries.ts]: Sorry, the change I had in mind is not possible at the moment. It has to wait until after assembling changes.

**Review by TorsteinHonsi — APPROVED** [review]
Thanks!
